modules/oxide/src/less/theme/content/placeholder/placeholder.less

```diff
@@ -18,11 +18,23 @@
 .mce-content-body:not([dir=rtl])[data-mce-placeholder] {
   &:not(.mce-visualblocks)::before {
     left: 1px;
+
+    @media (forced-colors: active) {
+      color: highlight;
+      filter: brightness(30%);
+      z-index: -1;
+    }
   }
 }
 
 .mce-content-body[dir=rtl][data-mce-placeholder] {
   &:not(.mce-visualblocks)::before {
     right: 1px;
+
+    @media (forced-colors: active) {
+      color: highlight;
+      filter: brightness(30%);
+      z-index: -1;
+    }
```

Comment:
Since these are the same, couldn't you make a new `data-mce-placeholder` rule that's not based on the `dir` attribute and avoid repeating the CSS?

Reply:
oh yeah, I got confused for a sec, now much better